backend/api.py

```diff
@@ -1,12 +1,54 @@
 # create our little application :)
-from flask import Flask
+import configparser
+from flask import Flask, request
+import tweepy
 
 app = Flask(__name__)
 
+def are_you_the_keymaster():
+    config = configparser.ConfigParser()
+    config.read('keys.ini')
+    return config
 
-@app.route('/opinion')
+
+def twitter_handler(message):
+    twconf = are_you_the_keymaster()['twitter']
+    auth = tweepy.OAuthHandler(twconf['conkey'], twconf['consec'])
+    auth.set_access_token(twconf['atok'], twconf['atoksec'])
+    api = tweepy.API(auth)
+    return str(api.update_status(message))
+    #return str(api.verify_credentials())
+
+
+def choose_handler():
+    # TODO: add other handlers
```

Comment:
Not to be implemented in this PR, but for this we can have each handler be in an array and then just randomly pick them! 

Fun